charts/linkerd2/templates/proxy-injector-rbac.yaml

```diff
@@ -76,6 +76,10 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   name: linkerd-proxy-injector-webhook-config
+  {{- if .Values.proxyInjector.injectCaBundle }}
+  annotations:
+    cert-manager.io/inject-ca-from: {{ printf "%s/%s" .Release.Namespace "linkerd-proxy-injector" }}
```

Comment:
`.Release.Namespace` works with Helm, but it'll break when using the CLI. Can you change it with `.Values.namespace`? I understand the former is the proper way, but I'll retrofit that later into #6635